README.md

```diff
@@ -1 +1,60 @@
-# Protfolio
\ No newline at end of file
+# Project Name
+
+> Here I am building a protfolio as part of my training and to be able to show my work to possible hiring managers, this website will contain all the projects that i will be working on with microverse and I will be using the following technologies.
+
+
+## Built With
+
+- HTML
+- CSS
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/shakerAbuDrais)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
+
```

Comment:
- [ ] Kindly make sure you update your README file by adding the **Project title, completing the Getting started section(with some information about cloning the project locally, setting up the app,...)** and removing the non-related sections to your project in the README file to make it professional. **I recommended you use [this template](https://github.com/tresorsawasawa/MyPortfolio/blob/develop/README.md) to help you fill all the sections**